astropy/modeling/tests/test_fitters.py

```diff
@@ -501,6 +501,15 @@ def test_param_cov(self):
         assert_allclose(fmod.parameters, beta.A.ravel())
         assert_allclose(olscov, fitter.fit_info['param_cov'])
 
+    def test_model_set_LevMar(self):
```

Comment:
It might also be nice to have a less trivial test (eg. comparing results with `LinearLSQFitter`), to prove that it comes up with the right numbers independently of its starting point and when each model has different statistics to minimize. I initially made a mistake in `FittingWithOutlierRemoval` where a test like this would have passed even though the results were wrong when the models differ.